R/plotting.R

```diff
@@ -48,14 +48,49 @@ names(.all_labs_values) <- .all_aes_names
   brush_set <- !is.null(brush_out$cmd)
 
   # Store the ggplot commands
-  cmds$todo[["ggplot"]] <- sprintf("ggplot(plot.data, %s) +", .build_aes(color = color_set))
-  cmds$todo[["point"]] <- "geom_point(size = 1.5) +"
+  cmds$todo[["ggplot"]] <- "ggplot() +"
+
+  # Implementing the brushing effect.
+  if (brush_set) {
+    brush_effect <- param_choices[[.brushEffect]]
+    if (brush_effect==.brushColorTitle) {
+      cmds$todo[["brush_color"]] <- sprintf(
```

Comment:
Does the order of `geom_point` calls have any effect? If it's anything like how base `points` behaves, I guess we should probably favour plotting of the brushed points on top of the other points.

Reply:
yep, the order matter, the last layer wins, i'll check again now

Reply:
argh.. obviously i coded it the wrong way around. fixing now